src/styles/homePage.css

```diff
@@ -0,0 +1,85 @@
+.classA {
+	display: flex;
+	flex-direction: column;
+	flex-wrap: wrap;
+}
+
+.quiz-container-bg {
+	background: var(--color-bg);
+	color: var(--color-text-1);
+}
+
+.card-container-bg {
+	background: var(--color-card-bg);
+	color: var(--color-text-1);
+}
+
+.abc {
+	display: flex;
+}
+
+.ab {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+}
+
+.ab1 {
+	display: flex;
+	justify-content: space-evenly;
+	align-items: center;
+}
+
+.quiz-img {
+	width: 100%;
+	border-radius: 0.5rem;
+}
+
+.shade {
+	box-shadow: var(--color-card-shadow);
+}
+
+.quiz-container {
+	width: 25rem;
+}
+
+.padding-small-card {
+	padding: 0.5rem 0.8rem 0.8rem 0.8rem;
+	border-radius: 0.5rem;
+}
+
+.card-selector-text {
+}
+
+.quiz-index-img {
+	opacity: 0.2;
+	width: 100%;
+	height: 42rem;
+}
+
+.container-absolule {
+	position: absolute;
+	top: 11%;
+	left: 25%;
+}
+
+.grid-col-4 {
+	display: grid;
+	grid-template-columns: repeat(4, 1fr);
+	background-color: var(
+		--Mak-color-bg-footer
+	);
+	color: var(--mak-color-txt-footer);
+	margin-top: 0.5rem;
+}
+
+.headerContainerMain {
+	background: var(
+		--color-card-bg
+	) !important;
+	color: var(--color-text-1) !important;
```

Comment:
do not use important as it will override user stylesheet

Reply:
I have imported properties form eCommerce and CL Projects they are overlapping that why i used it